packages/web-components/src/tabs/tabs.stories.ts

```diff
@@ -0,0 +1,235 @@
+import { html } from '@microsoft/fast-element';
+import { TabsOrientation } from '@microsoft/fast-foundation';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import type { Tabs as FluentTabs } from './tabs.js';
+import './define.js';
+import '../tab/define.js';
+import '../tab-panel/define.js';
+import { TabsAppearance as TabsAppearanceValues, TabsSize } from './tabs.options.js';
+
+type TabsStoryArgs = Args & FluentTabs;
+type TabsStoryMeta = Meta<TabsStoryArgs>;
+
+const defaultTabIds = ['first-tab', 'second-tab', 'third-tab', 'fourth-tab'];
+
+const tabsDefault = html`
+  <fluent-tabs
+    orientation=${component => component.orientation}
+    appearance=${component => component.appearance}
+    ?disabled=${component => component.disabled}
+    size=${component => component.size}
+    activeid=${component => component.activeid}
+  >
+    <fluent-tab id=${defaultTabIds[0]}> First Tab </fluent-tab>
+    <fluent-tab id=${defaultTabIds[1]}> Second Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[2]}> Third Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[3]}> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel id=${defaultTabIds[0] + '-panel'}>Tab One Content</fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[1] + '-panel'}>Tab Two Content</fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[2] + '-panel'}>Tab Three Content</fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[3] + '-panel'}>Tab Four Content</fluent-tab-panel>
```

Comment:
I'm going to nit on the stories here and the `id` usage. From a standards perspective, id's should be unique per document; these are all in the light DOM so it's kind of "bad practice" in that sense. From a tabs perspective, these _should_ get default id values if none is provided. Can we enumerate one set of id's for a story and then just not pass for the rest?